test/app-luatest/gh_4975_errors_in_init_scripts_are_stripped_test.lua

```diff
@@ -0,0 +1,20 @@
+local t = require('luatest')
+local justrun = require('test.justrun')
+
+local g = t.group('gh-4975')
+
+--[[
+Require a non-existent module with a long enough name to get an error message,
+which is longer than 1000 characters, like:
+
+LuajitError: module 'aaa<...>aaa' not found:
+    no file '/usr/local/lib/lua/5.1/aaa<...>aaa.so'
+    no file '/usr/local/lib/lua/5.1/override/aaa<...>aaa.so'
+    <...>
+]]
+g.test_long_init_error = function()
+    local module_name = string.rep('a', 500)
+    local args = {'-l', module_name, '-e', '""'}
+    local res = justrun.tarantool('.', {}, args, {stderr = true, nojson = true})
+    t.assert_gt(string.len(res.stderr), 1000, 'error is longer than 1000 chars')
```

Comment:
Nit: maybe at least match the error (`LuajitError: module 'aaa<...>aaa' not found`) with `assert_str_matches` for sanity?

Reply:
Done.